packages/terra-site/src/examples/action-footer/examples/action-footer/MultipleStartEndActions.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ActionFooter from 'terra-action-footer';
+import Button from 'terra-button';
+import Spacer from 'terra-spacer';
+
+const MultipleStartEndActionsExample = () => (
+  <ActionFooter
+    start={<a href="/#/site/action-footer">Start Action</a>}
+    end={
+      <div>
+        <Spacer isInlineBlock paddingRight="medium">
```

Comment:
UX Edits (2 of 3): 
To more closely follow common button placement, reverse the order so the first button is "Submit" and the second button is "Cancel"